.github/workflows/deploy-development.yml

```diff
@@ -0,0 +1,102 @@
+name: Deploy to development environment
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push events only for the main branch
+  push:
+    branches:
+      - main
+      - master
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# This workflow is made up of one job that calls the reusable workflow in graasp-deploy
+jobs:
+  graasp-deploy-ecs-backend-workflow:
+    # repository name
+    name: Graasp
+    uses: graasp/graasp-deploy/.github/workflows/cintegration-ecs-backend.yml@v1
```

Comment:
Do you also plan to rename the files in deploy ?

Reply:
At some point yes. But I think i'll refactor with the Github environments also when doing so.